packages/commerce-events-collectors/src/handlers/checkout/placeOrderAEP.ts

```diff
@@ -26,6 +26,9 @@ const aepHandler = async (event: Event): Promise<void> => {
                     shippingAmount: Number(orderContext.shipping?.shippingAmount) || 0,
```

Comment:
in light of the error handling changes, we should change this to `orderContext?.shipping?.shippingAmount)` -- i assume it will be there if you reconcile the branches, just adding a reminder :)